app/react-components/uikit/Tooltip.js

```diff
@@ -0,0 +1,26 @@
+// Vendor
+import React, { PropTypes } from 'react';
+
+const Tooltip = ({ innerHTML, children }) => (
+  <span 
+    onMouseOver={() => { 
+      let tooltip = document.querySelector('.global-tooltip')
+      tooltip.innerHTML = innerHTML;
+      tooltip.className = 'global-tooltip active';
```

Comment:
Might what to switch this to classList. Probably there will never be another class on the tooltip, but just in case :)